release/package-versions.json

```diff
@@ -215,44 +215,44 @@
     "version": "2.42.0"
   },
   "gitaly": {
-    "name": "gitaly-16.7.7",
+    "name": "gitaly-16.9.3",
     "pname": "gitaly",
-    "version": "16.7.7"
+    "version": "16.9.3"
   },
   "github-runner": {
-    "name": "github-runner-2.314.1",
+    "name": "github-runner-2.315.0",
     "pname": "github-runner",
-    "version": "2.314.1"
+    "version": "2.315.0"
   },
   "gitlab": {
-    "name": "gitlab-16.7.7",
+    "name": "gitlab-16.9.3",
     "pname": "gitlab",
-    "version": "16.7.7"
+    "version": "16.9.3"
   },
   "gitlab-container-registry": {
-    "name": "gitlab-container-registry-3.90.0",
+    "name": "gitlab-container-registry-3.91.0",
     "pname": "gitlab-container-registry",
-    "version": "3.90.0"
+    "version": "3.91.0"
   },
   "gitlab-ee": {
-    "name": "gitlab-ee-16.7.7",
+    "name": "gitlab-ee-16.9.3",
     "pname": "gitlab-ee",
-    "version": "16.7.7"
+    "version": "16.9.3"
   },
   "gitlab-pages": {
-    "name": "gitlab-pages-16.7.7",
+    "name": "gitlab-pages-16.9.3",
     "pname": "gitlab-pages",
-    "version": "16.7.7"
+    "version": "16.9.3"
   },
   "gitlab-runner": {
-    "name": "gitlab-runner-16.7.0",
+    "name": "gitlab-runner-16.9.1",
     "pname": "gitlab-runner",
-    "version": "16.7.0"
+    "version": "16.9.1"
```

Comment:
gitlab is on 16.9.3, while the runner is "just" on 16.9.1. Is there a good reason for this and could this mismatch be problematic (or: how tightly-coupled are the runner and gitlab?)

Reply:
Gitlab says that they should have the same major.minor version, so 16.9 in this case. The patch version is usually not the same. gitlab-runner-16.9.1 is the current release from last week.